README.md

```diff
@@ -3,55 +3,76 @@
 ## Install
 
 - This assumes an environment is already set up for npm packages - if not, please use npm init following the steps at [https://docs.npmjs.com/getting-started/using-a-package.json](https://docs.npmjs.com/getting-started/using-a-package.json).
-- run `npm install @patternfly/patternfly-next --save`
+- run `npm install @patternfly/patternfly-next`
 
-When you install PatternFly Next, the package includes:
+Any of the files below are meant for use in consuming PatternFly's styles. The recommended consumption approach will vary from project to project.
 
-- a single file for the entire compiled library: `node_modules/@patternfly/patternfly-next/patternfly.css`
-- individual files with each component compiled separately: `node_modules/@patternfly/patternfly-next/<ComponentName>/styles.css`
-- a single file for the entire library's source (SASS): `node_modules/@patternfly/patternfly-next/patternfly.scss`
-- individual files for each component's source (SASS): `node_modules/@patternfly/patternfly-next/<ComponentName>/styles.scss`
+- a single file for the complete PatternFly styles:
 
-Any of the files above are meant for use in consuming the library. The recommended
-consumption approach will vary from project to project.
+  ```
+  node_modules/@patternfly/patternfly-next/patternfly.css
+  ```
+
+- individual stylesheets for each of PatternFly's blueprints (e.g. components, layout):
+
+  ```
+  node_modules/@patternfly/patternfly-next/<BlueprintType>/<BlueprintName>/<blueprint-name>.css
+  ```
+
+- the base source file for PatternFly's styles ([SASS](https://sass-lang.com/)):
+
+  ```
+  node_modules/@patternfly/patternfly-next/patternfly.scss
+  ```
+
+- individual source files for each PatternFly blueprint (SASS):
+
+  ```
+  node_modules/@patternfly/patternfly-next/<BlueprintType>/<BlueprintName>/<blueprint-name>.css
```

Comment:
Thanks for catching that! Also, cool that's the first time I've gotten one of the new "suggestions". Amending that last commit rather than adding a new one via the PR suggestions' interface